crates/autopilot/src/database/onchain_order_events/ethflow_events.rs

```diff
@@ -94,31 +116,46 @@ impl OnchainOrderParsing<EthFlowData, EthFlowDataForDb> for EthFlowOnchainOrderP
             .context("append_ethflow_orders failed during appending pre_interactions")
     }
 
-    fn customized_event_data_for_event_index(
+    async fn customized_event_data_for_event_index(
         &self,
         event_index: &EventIndex,
+        quote: &shared::order_quoting::Quote,
         order: &Order,
         hashmap: &HashMap<EventIndex, EthFlowData>,
         _onchain_order_placement: &OnchainOrderPlacement,
-    ) -> EthFlowDataForDb {
-        EthFlowDataForDb {
+    ) -> Result<EthFlowDataForDb> {
+        let slippage = BigRational::new(
+            quote.data.quoted_buy_amount.to_big_int(),
+            order.buy_amount.to_bigint().unwrap(),
+        )
+        .to_f64()
+        .unwrap_or(f64::NAN);
+        let unix_timestamp_of_block = self
```

Comment:
If this call returns an error or an time-out, then we have the following error propagation:

```
customized_event_data_for_event_index
=>extract_custom_and_general_order_data
=>replace_events
=>update_events_from_latest_blocks 
-> and it does not update the last_handled_block. Hence, we will retry again to index the same orders in the next maintenance loop
=>run_maintenance_for_block_stream
-> and it will not update the last_handled_block in the metrics
```